app/Models/User.php

```diff
@@ -12,7 +12,8 @@ class User extends Authenticatable implements MustVerifyEmail
 {
     use HasFactory, Notifiable,SoftDeletes;
     const User ="User";
-    const Role="Role";
+    const Author="Author";
+    const isBanned=1;
```

Comment:
Constant naming format is wrong. Constant should be all uppercase.  Also, this seems like a boolean value. No need to create a constant for this.